examples/resolvers-planetscale/readme.md

```diff
@@ -0,0 +1,102 @@
+```shell
+CREATE TABLE Products (
+    id INT NOT NULL AUTO_INCREMENT,
+    name VARCHAR(255) NOT NULL,
+    slug VARCHAR(255) NOT NULL UNIQUE,
+    price INT NOT NULL DEFAULT 0,
+    onSale BOOLEAN DEFAULT FALSE,
```

Comment:
```suggestion
    on_sale BOOLEAN DEFAULT FALSE,
```

Reply:
I named it this because of a limitation Grafbase side so it would remove the need to transform and map this field.

Reply:
What's the limitation?

Reply:
```
[RuleError { locations: [Pos(2:3)], message: "Field \"on_sale\" is not in Camel lowercase, please use \"onSale\" instead." }]
```

Reply:
For the sake of the tutorial I wanted to keep a 1:1 mapping, if you think it's worth the explainer I'm happy to adjust though.

Reply:
All good, just trying to understand the limitation. You could perhaps do `SELECT onSale as 'on_sale'`?